decidim-admin/spec/forms/organization_form_spec.rb

```diff
@@ -12,6 +12,8 @@ module Admin
       let(:instagram_handler) { "My instagram awesome handler" }
       let(:youtube_handler) { "My youtube awesome handler" }
       let(:github_handler) { "My github awesome handler" }
+      let(:header_snippets) { "<my-html />" }
```

Comment:
Shouldn't we be checking this actually appears in the document?

Reply:
Done